### DISCUSSION TITLE
升级3.5以后打包产物的疑问？

### DISCUSSION THREAD

**Discussion by zhouzyc** [NervJS/taro#12224] [Q & A] [answered]
### 这个特性解决了什么问题？
编译后的文件名都是数字来标识，完全看不出来是什么
<img width="261" alt="WX20220801-133656@2x" src="https://user-images.githubusercontent.com/18420689/182079335-4c295798-de18-4a98-8e6a-1294a3a6e795.png">

### 这个 API 长什么样？
期望能够更加贴近业务实际的模块，让这个命名变得更加有意义，更方便的能够定位

**Comment by ZakaryCode** [comment, marked as answer]
根据提问内反馈截图，该问题是生产环境下编译 h5 平台，可以通过自行配置 chunkIds 优化 debug 体验

```js
webpackChain (chain) {
      chain.optimization.set('chunkIds', 'named')
}
```

**Reply by zhouzyc** [reply]
修改后，还是会有部分依旧是数字，基本是1000以下的数字，我找不到方法来对应数字具体是什么插件
<img width="1242" alt="WX20220825-093238@2x" src="https://user-images.githubusercontent.com/18420689/186553874-0e676696-3a1d-4651-8c2e-884dc00426a8.png">
